### DISCUSSION TITLE
Incident Forms AutoProcess

### DISCUSSION THREAD

**Discussion by Lyd1a6** [sheltermanager/asm3#1405] [Q&A / Help] [answered]
Hello!
We are trying to move our humane case reporting over to ASM and have it auto process to incidents. We are having issues with assigning the fields on the form to suspects rather than just the caller name and including a description field. Are there other commands or steps that I am missing to display these on new incidents?

**Comment by Tetzor** [comment, marked as answer]
Hi,

It is always assumed that the person completing the form would be the caller. Suspect data has to be added to the system by staff, it can't be added by a member of the public filling in an online form.

Kind regards,

Jon
